components/director/internal/domain/tenant/service.go

```diff
@@ -169,15 +174,42 @@ func (s *service) Update(ctx context.Context, id string, tenantInput model.Busin
 	return nil
 }
 
+func (s *service) GetCustomerIDParentRecursively(ctx context.Context, tenantID string) (string, error) {
```

Comment:
I'm just extracting the repo layer func to the service. So I guess it would be better to leave the naming as it is. But if you have strong feelings about this, I could rename the svc function.

Reply:
My point was that i don't care if it is "Recursively"  or not, but i am okey to leave it as it is.